lib/measures/tbd/README.md

```diff
@@ -19,44 +19,116 @@ ModelMeasure
 ## Arguments
 
 
+### Alter OpenStudio model (Apply Measures Now)
+For EnergyPlus simulations, leave CHECKED. For iterative exploration with Apply Measures Now, UNCHECK to preserve original OpenStudio model.
+**Name:** alter_model,
+**Type:** Boolean,
+**Units:** ,
+**Required:** false,
+**Model Dependent:** false
+
 ### Load 'tbd.json'
 Loads existing 'tbd.json' file (under '/files'), may override 'default thermal bridge' set.
 **Name:** load_tbd_json,
 **Type:** Boolean,
 **Units:** ,
-**Required:** true,
+**Required:** false,
 **Model Dependent:** false
 
 ### Default thermal bridge set
 e.g. 'poor', 'regular', 'efficient', 'code' (may be overridden by 'tbd.json' file).
 **Name:** option,
 **Type:** Choice,
 **Units:** ,
-**Required:** true,
+**Required:** false,
 **Model Dependent:** false
 
-### Alter OpenStudio model (Apply Measures Now)
-For EnergyPlus simulations, leave CHECKED. For iterative exploration with Apply Measures Now, UNCHECK to preserve original OpenStudio model.
-**Name:** alter_model,
+### Write 'tbd.out.json'
+Write out 'tbd.out.json' file e.g., to customize for subsequent runs (edit, and place under '/files' as 'tbd.json').
+**Name:** write_tbd_json,
 **Type:** Boolean,
 **Units:** ,
-**Required:** true,
+**Required:** false,
 **Model Dependent:** false
 
-### Write 'tbd.out.json'
-Write out 'tbd.out.json' file e.g., to customize for subsequent runs (edit, and place under '/files' as 'tbd.json').
-**Name:** write_tbd_json,
+### Uprate wall construction(s)
+Uprates selected wall construction(s), to meet overall Ut target
+**Name:** uprate_walls,
 **Type:** Boolean,
 **Units:** ,
-**Required:** true,
+**Required:** false,
+**Model Dependent:** false
+
+### Uprate roof construction(s)
+Uprates selected roof construction(s), to meet overall Ut target
+**Name:** uprate_roofs,
+**Type:** Boolean,
+**Units:** ,
+**Required:** false,
+**Model Dependent:** false
+
+### Uprate floor construction(s)
+Uprates selected floor construction(s), to meet overall Ut target
+**Name:** uprate_floors,
+**Type:** Boolean,
+**Units:** ,
+**Required:** false,
+**Model Dependent:** false
+
+### Wall Ut target (W/m2•K)
+Overall Ut target to meet for wall construction(s)
+**Name:** wall_ut,
+**Type:** Double,
+**Units:** ,
+**Required:** false,
+**Model Dependent:** false
+
+### Roof Ut target (W/m2•K)
+Overall Ut target to meet for roof construction(s)
+**Name:** roof_ut,
+**Type:** Double,
+**Units:** ,
+**Required:** false,
+**Model Dependent:** false
+
+### Floor Ut target (W/m2•K)
+Overall Ut target to meet for exposed floor construction(s)
+**Name:** floor_ut,
+**Type:** Double,
+**Units:** ,
+**Required:** false,
+**Model Dependent:** false
+
+### Wall construction(s) to 'uprate'
+Target 1x (or 'ALL') wall construction(s) to 'uprate'
```

Comment:
Maybe mention here that `wall_ut` must be set?

Reply:
The added "NONE" option should do the trick.